src/current/v23.2/log-sql-statistics-to-datadog.md

```diff
@@ -60,6 +60,17 @@ sinks:
       channels: [] # set to empty square brackets
 ~~~
 
+{{site.data.alerts.callout_success}}
+If you prefer to keep the `DD-API-KEY` in a file other than the `logs.yaml`, replace the `headers` parameter with the [`file-based-headers` parameter]({% link {{ page.version.version }}/configure-logs.md %}#file-based-headers):
+
+{% include_cached copy-clipboard.html %}
+~~~ yaml
+      file-based-headers: {DD-API-KEY: "{path/to/file}"} # replace with path of file containing DATADOG API key
+~~~
+
+The value in the file containing the Datadog API key can be updated without restarting the `cockroach` process. Send SIGHUP to the `cockroach` process to notify it to refresh the value.
```

Comment:
This would be a little clearer IMO if the second sentence started with something like

"Instead, send SIGHUP ..." 

to make the contrast with the previous sentence more obvious

Reply:
Added the "Instead" as a transition.